src/pages/UpSertUser/InsertPage/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { useHistory } from "react-router-dom";
+
+import useSnackbar from "src/components/SnackBars/useSnackbar";
+import { IUser } from "src/components/UserCard";
+
+import UpSertPage from "../UpSertPage";
+import useInsertUser from "../hooks/useInsertUser";
+
+const InsertPage = () => {
+    const history = useHistory();
+
+    const showSnackbar = useSnackbar();
+
+    const backToListPage = () => {
+        history.push("/user");
+    };
+
+    const showError = (message: string, type: "error" | "success") => {
+        showSnackbar({
+            color: type,
+            variant: "filled",
+            children: message,
+        });
+    };
+
+    const { mutate } = useInsertUser(backToListPage, showError);
+
+    const sendDataToServer = (user: IUser) => {
+        mutate({
+            name: user.name || "",
```

Comment:
I think we should validate instead of sending ""

Reply:
![image](https://user-images.githubusercontent.com/58205909/131253202-6d61bcda-8542-4ad4-b385-cd1c71e15106.png)